cypress/features/regression/designSystem/grid.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Design System Grid component
+  I want to test Design System Grid component
+
+  Background: Open Design System Grid component page
+    Given I open Design Systems basic "Grid" component docs page
+
+  @positive
+  Scenario: I confirm justify style
+    # When I click on component
```

Comment:
Uncomment this line please, if it is necessary

Reply:
fixed